test/units/testsuite-22.sh

```diff
@@ -5,6 +5,12 @@ set -o pipefail
 
 # shellcheck source=test/units/test-control.sh
 . "$(dirname "$0")"/test-control.sh
+# shellcheck source=test/units/util.sh
+. "$(dirname "$0")"/util.sh
+
+# Needed to write /usr/lib/tmpfiles.d to test /etc overrides in testsuite-22.13.sh
+maybe_mount_usr_overlay
```

Comment:
Is the overlay stuff still needed? I didn't follow the latest changes too closely, mainly https://github.com/systemd/systemd/pull/32567/, so I presume `/usr` is still read-only?

Reply:
Nope, I'll remove it everywhere